components/DateRangePicker.tsx

```diff
@@ -1,4 +1,4 @@
-import React from "react";
+`import React from "react";
```

Comment:
I assume this change was accidental?

```suggestion
import React from "react";
```